METADATA_SUPPORT.md

```diff
@@ -510,6 +485,36 @@ v55 introduces the following new types.  Here's their current level of support
 
 |Metadata Type|Support|Notes|
 |:---|:---|:---|
+|AssessmentQuestion|❌|Not supported, but support could be added|
+|AssessmentQuestionSet|❌|Not supported, but support could be added|
+|BotTemplate|❌|Not supported, but support could be added|
+|CallCtrAgentFavTrfrDest|❌|Not supported, but support could be added|
+|CustomAddressFieldSettings|✅||
+|DataImportManagementSettings|✅||
+|DecisionMatrixDefinition|✅||
+|DecisionMatrixDefinitionVersion|✅||
+|DigitalExperience|undefined|undefined|
```

Comment:
Seems odd that `undefined` is the documented value for `Support` and `Notes`. Is that intentional/

Reply:
at the time this PR was generated, looks like DigitalExperience in v55 was missing some values.  They've got it fixed now https://na45.test1.pc-rnd.salesforce.com/mdcoverage/api.jsp

Reply:
nope, good catch.  There's a another scenario I didn't have (item is not in registry, support could be added but not for tracking)